_docs/overview/key-concepts.md

```diff
@@ -9,23 +9,31 @@ description: Overview of key concepts leveraged by Glacier to provide
 The process of using a private key to access bitcoins, referred to as "signing
 a transaction". 
 
-## Private keys
+## Private key
 
-Bitcoin balances are stored in the Bitcoin blockchain, a global
-decentralized ledger, which can be unlocked with a piece of information
-known as a "private key". This is analogous to storing coins in a
-padlocked box, of which there can be multiple padlocks - see the 
-[Multisignature Security](/docs/overview/multi-signature-security)
- section.
+A 256-bit number, usually expressed in 64-character hexadecimal, which can unlock
```

Comment:
Adding to my earlier comment about hexadecimal...the keys created by Glacier are not hex but base58 WIF. Even more reason to relegate such details to a footnote.

Reply:
Agreed, it is too complex. I'll make a new commit changing it to

"usually expressed as a 64-character string of numbers and letters"

and we can add a footnote later, once the functionality is available.

Reply:
Except that WIF is not 64 characters long.